xml_schema_derive/src/xsd/element.rs

```diff
@@ -103,7 +103,8 @@ impl Element {
     context: &XsdContext,
     prefix: &Option<String>,
   ) -> TokenStream {
-    if self.name.is_empty() {
+    if self.name.is_empty() && (self.refers.is_none() || matches!(self.refers.as_deref(), Some("")))
```

Comment:
it may relevant to create a non public method to perform:
```rust
fn get_refers(&self) -> Option<String> {
  self.refers.and_then(|refers| {
     if refers.is_empty() {
       None
     } else {
       Some(refers.to_owned())
     }
  })
}
```

Reply:
Thanks for the review.
I refactored the code to use `get_refers`, but currently it is not using `String`. So I changed it to `str`.